src/main/java/com/zaxxer/nuprocess/linux/ProcessEpoll.java

```diff
@@ -48,9 +47,12 @@
 
    static
    {
-      eventPool = new ArrayBlockingQueue<EpollEvent>(EVENT_POOL_SIZE);
+      eventPool = new ArrayBlockingQueue<>(EVENT_POOL_SIZE);
       for (int i = 0; i < EVENT_POOL_SIZE; i++) {
-         eventPool.add(new EpollEvent());
+         EpollEvent event = new EpollEvent();
+         // disable auto-read for these events, because they're only used to call epoll_ctl
+         event.setAutoRead(false);
```

Comment:
This reduces `SharedMemory` allocations in `registerProcess` when (needlessly) copying from native memory to heap memory after calling `epoll_ctl`